index.js

```diff
@@ -36,7 +42,7 @@ module.exports = function(str, options) {
   }).join(newline);
 
   if (options.trim === true) {
-    result = result.replace(/[ \t]*$/gm, '');
```

Comment:
I guess `/\s+$/gm` works faster

Reply:
Thanks, let me check on that.

Reply:
maybe or native better regex can be faster. need to bench to check that

Reply:
Hi @aashutoshrathi, sorry for the long delay, and thanks for the PR. I'll try to get this merged in ASAP. 

> I guess `/\s+$/gm` works faster

Yes, let's use this pattern without the `m` flag. Using `+` will also be a bit safer since it won't allow that pattern to match an empty string.

Reply:
`/[ \t]+$/g` is slightly better than `/\s+$/g` it would seem. Unfortunately both regexes are suspectable to redos, as opposed to `trimEnd()`;

`$ time node -e 'payloadstr=""; for ( i=0; i < 100000; i++ ) payloadstr+="\t"; payloadstr+="a"; payloadstr.replace(/[ \t]+$/gm,"");'`
`real    0m21.084s`
`$ time node -e 'payloadstr=""; for ( i=0; i < 100000; i++ ) payloadstr+="\t"; payloadstr+="a"; payloadstr.replace(/\s+$/g,"");'`
`real    0m8.587s`
`$ time node -e 'payloadstr=""; for ( i=0; i < 100000; i++ ) payloadstr+="\t"; payloadstr+="a"; payloadstr.replace(/[ \t]+$/g,"");'`
`real    0m6.169s`
`$ time node -e 'payloadstr=""; for ( i=0; i < 100000; i++ ) payloadstr+="\t"; payloadstr+="a"; payloadstr.trimEnd();'
real    0m0.082s`

Reply:
@jonschlinkert Any update when this PR will be merged ?